frontend/public/components/data-view-poc/DataViewPodList.tsx

```diff
@@ -0,0 +1,801 @@
+      column.props.sort.onSort = onSort;
+    }
+  });
+  return { dataViewRows, dataViewColumns, pagination };
+}
+
+// Mostly copied over from public/components/filter-toolbar.tsx
+// Only taken the labels filter as it is unique to OCP and not in data view
+// Seems like it could be generalized and transferred as a new filter type to data view
+const LabelsFilter = ({
+  onFilterChange,
+  data,
+  title,
+  filterId,
+  showToolbarItem,
+}: {
+  onFilterChange: OnFilterChange;
+  data: PodKind[];
+  filterId: string;
+  title: string;
+  showToolbarItem?: boolean;
+}) => {
+  const { t } = useTranslation();
+  const [labelInputText, setLabelInputText] = React.useState('');
+  const [searchParams] = useSearchParams();
+
+  const [labelSelection, onLabelSelectionChange] = useLabelSelectionFix(searchParams, filterId);
+  const applyLabelFilters = (values: string[]) => {
+    setLabelInputText('');
+    onLabelSelectionChange(values);
+    onFilterChange(filterId, { all: values });
+  };
+
+  return (
+    <ToolbarFilter
+      deleteLabelGroup={() => {
+        setLabelInputText('');
+        applyLabelFilters([]);
+      }}
+      labels={labelSelection}
+      deleteLabel={(f, chip: string) => {
+        setLabelInputText('');
+        applyLabelFilters(_.difference(labelSelection, [chip]));
+      }}
+      categoryName={title}
+      showToolbarItem={showToolbarItem}
+    >
+      <div className="pf-v6-c-input-group co-filter-group">
+        <AutocompleteInput
+          className="co-text-node"
+          onSuggestionSelect={(selected) => {
+            applyLabelFilters(_.uniq([...labelSelection, selected]));
+          }}
+          showSuggestions
+          textValue={labelInputText}
+          setTextValue={setLabelInputText}
+          placeholder={t('public~Search by label...')}
+          data={data}
+        />
+      </div>
+    </ToolbarFilter>
+  );
+};
+
+type DataViewPodListProps = {
+  loaded: boolean;
+  data: PodKind[];
+  showNodes?: boolean;
+  filters?: RowFilter<PodKind>[];
+  onFilterChange: OnFilterChange;
+  columnLayout?: ColumnLayout;
+  showNamespaceOverride?: boolean;
+};
+
+const DataViewPodList = ({
+  data,
+  showNodes,
+  loaded,
+  filters,
+  onFilterChange,
+  columnLayout,
+  showNamespaceOverride,
+}: DataViewPodListProps) => {
+  const { t } = useTranslation();
+  const { dataViewColumns, dataViewRows, pagination } = useDataViewData({
+    data,
+    showNodes,
+    showNamespaceOverride,
+  });
+  const [searchParams, setSearchParams] = useSearchParams();
+  const initialFilters = filters?.reduce((acc, curr) => {
+    acc[curr.filterGroupName] = searchParams.getAll(curr.filterGroupName);
+    return acc;
+  }, {});
+  const debouncedOnFilterChange = useDebounceCallback(onFilterChange, 500);
+  const debouncedSetSearchParams = useDebounceCallback(setSearchParams, 500);
+  // Currently there is a big that will only return the first item from the query, even though there are multiple items for one group
```

Comment:
@Hyperkid123, can you clarify what you mean here?  I'm not clear on what you're describing.

Reply:
Oh sorry. There was a bug. I fixed it here: https://github.com/patternfly/react-data-view/pull/395

Bug not big 🙂